src/jackett.py

```diff
@@ -113,30 +118,35 @@ def filter_results(method, results):
 
     if get_setting('filter_keywords_enabled', bool):
         results = filter.keywords(results)
-        log.debug(f"results after filtering keywords: {results}")
+        log.info(f"filtering keywords {len(results)}")
+        log.debug(f"filtering keywords results: {results}")
 
     if get_setting('filter_size_enabled', bool):
         results = filter.size(method, results)
-        log.debug(f"results after filtering size: {results}")
+        log.info(f"filtering size {len(results)}")
+        log.debug(f"filtering size results: {results}")
 
     if get_setting('filter_include_resolution_enabled', bool):
         results = filter.resolution(results)
-        log.debug(f"results after filtering resolution: {results}")
+        log.info(f"filtering resolution {len(results)}")
+        log.debug(f"filtering resolution results: {results}")
 
     if get_setting('filter_include_release', bool):
         results = filter.release_type(results)
-        log.debug(f"results after filtering release type: {results}")
+        log.info(f"filtering release type {len(results)}")
+        log.debug(f"filtering release type {len(results)} results: {results}")
 
     if get_setting('filter_exclude_no_seed', bool):
         results = filter.seed(results)
-        log.debug(f"results after filtering no seeds: {results}")
+        log.info(f"filtering no seeds {len(results)}")
+        log.debug(f"filtering no seeds results: {results}")
 
-    results = filter.unique(results)
-    log.debug(f"results after filtering for unique items: {results}")
+    # results = filter.unique(results)
+    # log.debug(f"filtering for unique items {len(results)} results: {results}")
```

Comment:
I'd rather keep the unique call here as this does the filtering. Any reason this has moved?

Reply:
yes, `filter.unique()` can be called only after magnet resolving. 
Now i make resolving after filtering, so i will have less links to resolve. Because it takes lots of time. 
And only after i call `filret.unique()`, from `search_jackett()`

Reply:
done